src/app/(frontend)/api/stripe/webhooks/route.ts

```
@@ -0,0 +1,87 @@
+          }
+
+          const expandedSession = await stripe.checkout.sessions.retrieve(data.id, {
+            expand: ['line_items.data.price.product']
+          })
+
+          if (!expandedSession.line_items?.data || !expandedSession.line_items?.data.length) {
+            throw new Error('Webhook error: Line items not found')
+          }
+          const lineItems = expandedSession.line_items.data as ExpandedLineItem[]
+
+          for (const item of lineItems) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)